### ISSUE TITLE
Stylemixing specific layers

### ISSUE THREAD

**Issue by nmounsef** [genforce/idinvert_pytorch#34]
Is it possible to choose which layers to stylemix instead of choosing just the starting layer to the final layer?

**Comment by zhujiapeng** [comment]
Yes, in `mix_style.py` of line 67, just give the specific layer to the `mix_layers `
